rasa/nlu/test.py

```diff
@@ -451,15 +455,10 @@ def evaluate_response_selections(
             confusion_matrix_filename = os.path.join(
                 output_directory, confusion_matrix_filename
             )
-        _labels = [
-            response_to_intent_target[label]
-            if label in response_to_intent_target
-            else f"'{label[:20]}...' (response not present in test data)"
-            for label in labels
-        ]
+
         plot_utils.plot_confusion_matrix(
             confusion_matrix,
-            classes=_labels,
+            classes=labels,
```

Comment:
So `labels` is now always the retrieval intent?

Reply:
Yes, it makes the files look much cleaner IMO and conveys all the information also anyways.